README.md

```diff
@@ -1,4 +1,231 @@
+# acf-field-group-composer
+
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
 [![Build Status](https://img.shields.io/travis/bleech/acf-field-group-composer.svg?style=flat-square)](https://travis-ci.org/bleech/acf-field-group-composer)
-<!-- [![Style Check](https://styleci.io/repos/73226024/shield)](https://styleci.io/repos/73226024) -->
 [![Code Quality](https://img.shields.io/scrutinizer/g/bleech/acf-field-group-composer.svg?style=flat-square)](https://scrutinizer-ci.com/g/bleech/acf-field-group-composer)
 [![Code Coverage](https://img.shields.io/coveralls/bleech/acf-field-group-composer.svg?style=flat-square)](https://coveralls.io/github/bleech/acf-field-group-composer)
+
+> Configuration builder for advanced custom fields field groups
+
+This plugin helps you create reusable ACF fields and field groups with code.
+
+## Table of Contents
+
+- [Background](#background)
+- [Install](#install)
+- [Usage](#usage)
+- [API](#api)
+- [Maintainers](#maintainers)
+- [Contribute](#contribute)
+- [License](#license)
+
+## Background
+
+ACF provides a great interface to create custom meta boxes in WordPress. However, there are to major downsides using it out of the box.
```

Comment:
"out of the box:"

I would then start a real list below of the two downsides, rather than "first" and "second".